spec/new-productions.htm

```diff
@@ -9,6 +9,18 @@
   </emu-grammar>
 </emu-clause>
 
+<emu-clause id="public-field-definition-early-errors">
+  <h1>Static Semantics: Early Errors</h1>
+    PublicFieldDefinition :
+      PropertyName[?Yield] Initializer?
+  <emu-grammar>
+  </emu-grammar>
+  <emu-alg>
+    1. It is a SyntaxError if |Initializer| contains |NewTarget|.
```

Comment:
Note that advancement right now means Stage2 -- which doesn't require finalized spec text (i.e. spec text can have TODOS and require editorial work so long as the major aspects of it are in place)

But of course we should address your concerns: The purpose here is to specify that it is a static error if `new.target` or `arguments` is syntactically present in the initializer position. Is there more context around your misunderstanding you can provide?